src/Container/Argument/ArgumentResolverInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+declare(strict_types=1);
+
+namespace Cake\Container\Argument;
+
+use Cake\Container\ContainerAwareInterface;
+use ReflectionFunctionAbstract;
+
+interface ArgumentResolverInterface extends ContainerAwareInterface
+{
+    /**
+     * @param array $arguments
```

Comment:
```suggestion
     * @param array<LiteralArgument> $arguments
```
I guess

Reply:
phpstan told me it can be `LiteralArgument|ResolvableArgument`